.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.5, 3.6, 3.7, pypy3]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Lint with pylint
+      if: matrix.python == '3.7'
+      run: ./lint.sh all
+    - name: Test with pytest
+      if: success() || failure()
+      run: pytest
+    - name: Set up Node.js 10
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - name: Run integration tests against emulator
+      run: |
+        npm install -g firebase-tools
```

Comment:
I think we should use https://firebase.tools instead so that we can drop the setup-node action and build faster, but I think this is fine for now as well. Also, have we considered caching `~/.cache/firebase` which contains the emulator and (later) the standalone CLI?

Reply:
Been experimenting with this, but firebase.tools absolutely refuses to work in the GitHub Actions environment (files a bug for the CLI for this). There are bunch of workarounds I can use to get this to work -- which involves pulling the CLI binary for Linux explicitly. But I'd rather do all that in a separate PR.